src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -111,6 +115,17 @@ def check_run_conversion_with_backend(self, nwbfile_path: str, backend: Literal[
             **self.conversion_options,
         )
 
+    def check_configure_backend_for_equivalent_nwbfiles(self, backend: Literal["hdf5", "zarr"] = "hdf5"):
+        metadata = self.interface.get_metadata()
+        if "session_start_time" not in metadata["NWBFile"]:
+            metadata["NWBFile"].update(session_start_time=datetime.now().astimezone())
+
+        nwbfile_1 = self.interface.create_nwbfile(metadata=metadata, **self.conversion_options)
+        nwbfile_2 = self.interface.create_nwbfile(metadata=metadata, **self.conversion_options)
+
+        backend_configuration = get_default_backend_configuration(nwbfile=nwbfile_1, backend=backend)
+        configure_backend(nwbfile=nwbfile_2, backend_configuration=backend_configuration)
```

Comment:
What are we testing here for? I am looking for an assertion, is it just that it does not fail when you try to re-map? In other words, how would this test fai?

Reply:
Test is if it doesn't error, like the other checks in the mixin final to file write

It would fail if some object in the `backend_configuration` was not in the `nwbfile_2` or had any other problem throughout the process

Reply:
I see, do you think that we would like to add a test where this fails to catch that it fails in the way that we intend it to fail (a location in the nwbfile is not present in the backend configuration)?

Reply:
Added tests